module-types/Datacom/PSU-600-DC-F.yaml

```diff
@@ -0,0 +1,10 @@
+---
+manufacturer: Datacom
+model: PSU 600 DC-F
+part_number: 800.5257.xx
+comments: DC power supply with -48 to -60VDC input and the ability to provide 600W of power. Features forced ventilation with airflow exiting from the power
+  supply.
+power-ports:
+  - name: PSU
```

Comment:
Please update this name to use the `{module}` definition

Reply:
sorry, but I don't understand how it should be